clients/admin-ui/src/features/system/VendorSelector.tsx

```diff
@@ -130,165 +134,185 @@ const VendorSelector = ({
+        name: true,
+      },
+      // only validate if nothing is typed in the select's search input to
+      // prevent incorrect "required field" error like above
+      !searchParam
+    );
+  };
+
+  // we have to build the typeahead from scratch, too much context-specific
+  // is needed to use the existing CustomCreatableSelect component
+  const typeaheadSelect = (
+    <FormControl isInvalid={isInvalid} isRequired width="100%">
+      <VStack alignItems="start" position="relative" width="100%">
+        <HStack spacing={1}>
+          <Label htmlFor="name" fontSize="xs" my={0} mr={1}>
+            System name
+          </Label>
+          <QuestionTooltip label="Enter the system name" />
+        </HStack>
+        <Box width="100%" data-testid="input-name">
+          <CreatableSelect
+            name="name"
+            options={suggestions}
+            isRequired
+            value={selected}
+            onBlur={handleBlur}
+            onChange={(newValue, actionMeta) =>
+              handleSelectChange(newValue, actionMeta)
+            }
+            onInputChange={(e) => setSearchParam(e)}
+            inputValue={searchParam}
+            size="sm"
+            onKeyDown={(e) => {
+              if (e.key === "Tab") {
+                handleTabPressed();
+              }
+            }}
+            tabSelectsValue={hasVendorSuggestions}
+            classNamePrefix="custom-select"
+            placeholder="Enter system name..."
+            instanceId="select-name"
+            isDisabled={!isCreate && lockedForGVL}
+            menuPosition="absolute"
+            isSearchable
+            isClearable={!!searchParam}
+            focusBorderColor="primary.600"
+            formatCreateLabel={(inputValue) =>
+              `Create new system "${inputValue}"...`
+            }
+            chakraStyles={{
+              container: (provided) => ({
+                ...provided,
+                flexGrow: 1,
+                backgroundColor: "white",
+              }),
+              option: (provided, state) => ({
+                ...provided,
+                background:
+                  state.isSelected || state.isFocused ? "gray.50" : "unset",
+              }),
+              dropdownIndicator: (provided) => ({
+                ...provided,
+                display: "none",
+              }),
+              indicatorSeparator: (provided) => ({
+                ...provided,
+                display: "none",
+              }),
+            }}
+            components={{ Option: CustomDictOption }}
+          />
+        </Box>
+        <ErrorMessage
+          isInvalid={isInvalid}
+          message={meta.error}
+          fieldName="name"
         />
-      </HStack>
-      {fieldsSeparated ? (
-        <DictSuggestionTextInput
+        <Text
+          aria-hidden
+          position="absolute"
+          backgroundColor="transparent"
+          style={{ marginTop: "31.52px", marginLeft: "13px" }}
```

Comment:
For some reason, these margins don't get applied when using the Chakra style props, so I did it this way instead.